homeassistant/components/advantage_air/sensor.py

```diff
@@ -48,19 +48,12 @@ class AdvantageAirTimeTo(AdvantageAirEntity, SensorEntity):
 
     def __init__(self, instance, ac_key, action):
         """Initialize the Advantage Air timer control."""
-        super().__init__(instance, ac_key)
-        self.action = action
-        self._time_key = f"countDownTo{self.action}"
-
-    @property
-    def name(self):
-        """Return the name."""
-        return f'{self._ac["name"]} Time To {self.action}'
-
-    @property
-    def unique_id(self):
-        """Return a unique id."""
-        return f'{self.coordinator.data["system"]["rid"]}-{self.ac_key}-timeto{self.action}'
+        super().__init__(instance, ac_key, action)
```

Comment:
```suggestion
        super().__init__(instance, ac_key)
```
**action** shouldn't be passed as the **zone_key** of AdvantageAirEntity

Reply:
@tkdrob could you please include this suggestion, as I suspect it may lead to a key error.